content/en/learn.md

```diff
@@ -26,7 +26,7 @@ There's a ton of information about NumPy out there. If you are just starting, we
 
 <i class="fas fa-book"></i> **Books**
 
-* [Guide to NumPy *by Travis E. Oliphant*](https://web.mit.edu/dvp/Public/numpybook.pdf) This is a free version 1 from 2006. For the latest copy (2015) see [here](https://dl.acm.org/doi/10.5555/2886196).
+* [Guide to NumPy *by Travis E. Oliphant*](https://web.mit.edu/dvp/Public/numpybook.pdf) This is the first edition of the book published in 2006. For the latest edition released in 2015, [click here](https://dl.acm.org/doi/10.5555/2886196).
```

Comment:
I think you should mention the "free" part. You can probably break the line after the first link if you keep the indentation.

Reply:
Folks naturally assume books cost money unless it is specifically mentioned that they are free. Maybe break this into two entries, one for each book. You could put free in parenthesis in the first one.